src/react-redux/__tests__/with-api-client.spec.js

```diff
@@ -227,5 +237,95 @@ describe( 'withApiClient', () => {
 			const ComponentWithApiClient = withApiClient( 'test', { mapSelectorsToProps, getClientKey } )( Component );
 			mount( <ComponentWithApiClient clientKey="123" />, { context: { getApiClient } } );
 		} );
+
+		it( 'should render without mapSelectorsToProps', () => {
+			const ComponentWithApiClient = withApiClient( 'test', { getClientKey } )( Component );
+
+			const doRender = () => {
+				mount( <ComponentWithApiClient clientKey="123" />, { context: { getApiClient } } );
+			};
+
+			expect( doRender ).not.toThrow();
+		} );
+	} );
+
+	describe( '#mapMutationsToProps', () => {
+		it( 'should call mapMutationsToProps if present.', () => {
+			mapMutationsToProps = jest.fn();
+
+			const ComponentWithApiClient = withApiClient( 'test', { mapMutationsToProps, getClientKey } )( Component );
+			mount( <ComponentWithApiClient clientKey="123" />, { context: { getApiClient } } );
+
+			expect( mapMutationsToProps ).toHaveBeenCalledTimes( 1 );
+		} );
+
+		// TODO: This test runs code a bit deep, split it up into an ApiClient test and this one.
```

Comment:
It is a bit deep of a test, but pretty nice to see it all written out here - great example code.